### ISSUE TITLE
The /locate command is finding different Veins than actually generate (Custom Biome-filtered Veins)

### ISSUE THREAD

**Issue by Tachrayonic** [tom5454/Create-Ore-Excavation#86]
tldr; The issue is that the /locate command is finding different Veins than actually generate, when whitelisting by the underground Biomes.

For my friend's server, I've set up custom Veins.
I set up pairs of identical veins. (eg. Rich Volcanic and Volcanic)
These Veins would have the same spread, separation and salt to intentionally collide them.
The "Rare" version of the Vein has higher priority, but is whitelisted to certain Biomes.

For the most part, this works perfectly. However, it is a bit inconsistent when it comes to underground Biomes (Lush Caves, Dripstone Caves, Deep Dark). In these cases, we would find that attempting to find a Rare vein using the locate command would often point us to a common vein, but this common vein would have an underground Biome below them.

My question here then, is how does the /locate command sample the Biomes in the target chunk, and how does that differ to how the Veins actually generate?

**Comment by tom5454** [comment]
It pick a random block position in the chunk based on the world seed.

**Comment by Tachrayonic** [comment]
Ah, very straightforward.

The only issue then, is that the /locate and the actual generation are finding different veins. Maybe they aren't using the same seed?
